tox.ini

```diff
@@ -25,3 +27,5 @@ deps =
     flake8
     -rdev-requirements.txt
 commands=flake8 script_runner
+
+[testenv:pywin]
```

Comment:
when you specify a configuration on tox like this, does it mean it inherits all configuration from `testenv`?